external/iplib/aifc.py

```
@@ -0,0 +1,921 @@
+def _read_ushort(file):
+    try:
+        return struct.unpack('>H', file.read(2))[0]
+    except struct.error:
+        raise EOFError
+
+def _read_string(file):
+    length = ord(file.read(1))
+    if length == 0:
+        data = b''
+    else:
+        data = file.read(length)
```

Ruff (0.8.2)

Use ternary operator `data = b"" if length == 0 else file.read(length)` instead of `if`-`else`-block

Replace `if`-`else`-block with `data = b"" if length == 0 else file.read(length)`

(SIM108)